example/CMakeLists.txt

```diff
@@ -24,5 +24,5 @@ if(cxx_modules)
 else()
   set(app_src main_fallback.cc)
 endif()
-add_executable(main ${app_src})
+add_stdmodular_executable(main ${app_src})
```

Comment:
I think it better be a flag for `add_executable`, e.g. `STD_HEADER_UNITS`.

Reply:
Nevermind, `add_executable` is a standard function.

Reply:
yes, and also I didn't find any way to overload/replace `add_executable` that lets you use `add_executable` having a parameter when set to, for example, `STD_HEADER_UNITS` or or `STD_HEADER_IMPORTS` will use a wrapper around the function by the same name (but with different arguments), but by default CMake only sees the overloaded function and the original function with the same name, and will result in a recursive call. 

It could have been possible to do away with some tricks to `cmake_parse_arguments()`, but I don't know how.

Reply:
Having a separate function makes sense. I suggest renaming it to `add_module_executable` for consistency with `add_module_library`.